master/internal/config/config.go

```diff
@@ -111,6 +120,10 @@ func DefaultConfig() *Config {
 				RsaKeySize: 1024,
 			},
 			AuthZ: *DefaultAuthZConfig(),
+			Token: TokenConfig{
+				MaxLifespanDays:     DefaultTokenMaxLifespanDays,
```

Comment:
I don't think our default should be infinite. Let's make it 30 days and let folks opt-in to these wild infinite lifespan tokens.